i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-hide-scrollbars.mdx

```diff
@@ -0,0 +1,63 @@
+
+## Problem {#problem}
+
+One of the reasons for test failures when testing layouts using screenshots is the presence of scrollbars in the browser at the moment the screenshot is taken. There can be 3 scenarios where a diff occurs between the reference screenshot and the current one taken during the test run:
+
+1. There was no scrollbar when the reference screenshot was taken, but it appeared when taking the current screenshot.
+2. There was a scrollbar when the reference screenshot was taken, but it did not appear in time when taking the current screenshot.
+3. There is a scrollbar on both the reference and current screenshots, but the positions, sizes, or transparency of the scrollbars differ due to the timing of when the screenshots were taken relative to the appearance of the scrollbar.
+
+The screenshot below shows an example corresponding to the first scenario:
+
+![diff due to scrollbar](/img/docs/guides/how-to-hide-scrollbars.diff-in-screenshot.png)
+
+## Solution 1: screenshotDelay {#solution_1}
+
+In testplane settings, there is a mandatory option `browsers` that specifies the set of browsers available in the project and their properties. Select the browser where you are experiencing diffs due to scrollbars and add the `screenshotDelay` option for it:
+
+```javascript
+module.exports = {
+    browsers: {
+        iphone: {
+            screenshotDelay: 600, // Delay before taking a screenshot in ms
+
+            // other browser settings...
+        },
+
+        // other browsers...
+    },
+
+    // other testplane settings...
+};
+```
+
+The `screenshotDelay` option sets a pause in milliseconds that testplane should wait before taking a screenshot (before executing the `assertView` command).
+
+### How can it help? {#how_can_it_help}
+
+Often screenshots generate diffs because the test needs to scroll the page to the required element just before taking the screenshot. After the scroll is performed, the scrollbar might still be visible on the screen for some time and thus, might appear in the screenshot if taken immediately. The `screenshotDelay` gives the scrollbar time to disappear.
+
+However, this method does not always work as it depends on the implementation and behavior of the browsers.
+
+## Solution 2: Disabling Scrollbars {#solution_2}
+
+If scrollbars appear in screenshots in the Chrome browser, they can be disabled using the [DevTools protocol][CDP].
+
+To do this, add the [hermione-hide-scrollbars][hermione-hide-scrollbars] plugin to your project and specify in its settings the list of browsers for which you want to disable scrollbars in the tests.
+
+<Admonition type="warning">
+    Update the Chrome browser to version 72.1 or higher for this functionality to work in your
+    tests. Earlier versions of Chrome do not support the _Emulation.setScrollbarsHidden_ command,
+    which is used to disable the scrollbars.
+</Admonition>
+
+## Keywords {#keywords}
+
+-   screenshotDelay
+-   hermione-hide-scrollbars
```

Comment:
Just curious: why do we need keywords? For CEO stuff? When i saw these, i instantly wanted to click it, but its just text :(

Reply:
They were there in the original docs, so I kept them in the translation. It's not clear to me either (I guess originally their purpose was to give you an idea what to look for), we may get rid of them in the future.